src/entercommand.js

```diff
@@ -84,3 +87,11 @@ function splitBlock( writer, splitPos ) {
 	writer.split( splitPos );
 	writer.setSelection( splitPos.parent.nextSibling, 0 );
 }
+
+function* getAllowedAttributes( schema, allAttributes ) {
```

Comment:
The name is slightly misleading IMO. The name suggest `schema.getAllowedAttributes()` use (well getting allowed attributes in conjunction with schema). But this methods returns attributes that are copied on enter.

I'd go with `getCopyOnEnterAttributes()`.